### PR TITLE
Add stat uncertainty and rebin options in plotter

### PR DESCRIPTION
**Pull request by jeyserma**
Added options to plot the stat uncertainty (sig, bkg, sig+bkg) and optional uniform rebinning

### CODE REVIEW

File: config/doPlots.py [hunk 1]
```diff
@@ -75,11 +87,11 @@ def mapHistos(var, label, sel, param):
     for b in hbackgrounds:
         if len(hbackgrounds[b])==0:
             hbackgrounds=removekey(hbackgrounds,b)
-
+
```
Comment (kjvbrt) [inline config/doPlots.py:90]: can you remove unnecessary spaces?

**Comment by kjvbrt** [comment]
Hello @jeyserma,

do you have any example plot?

**Comment by jeyserma** [comment]
An example of using the stat unc and rebinning is given in examples/FCCee/higgs/mH-recoil/mumu/analysis_plots.py:

plotStatUnc = True
rebin = [1, 1, 1, 1, 2] # uniform rebin per variable (optional)

**Comment by kjvbrt** [comment]
OK, I generated it :)

I find it a bit hard to read, maybe increase the pitch of the diagonal lines or use just solid grey?

@BrieucF What do you think?

leptonic_recoil_m_zoom2_ZH_stack_lin

**Comment by BrieucF** [comment]
Yes, the example I pointed in the FCCSW forum thread [1] is obtained with something like:

```
stat_and_syst->SetMarkerSize(0);
stat_and_syst->SetMarkerStyle(0);
stat_and_syst->SetFillStyle(3154);
stat_and_syst->SetFillColor(42);
stat_and_syst->Draw("E2 same");
```

Some inspiration could probably be taken from [2].

[1] https://fccsw-forum.web.cern.ch/t/histogram-uncertainties/154/2
[2] https://github.com/cp3-llbb/plotIt/blob/master/src/TH1Plotter.cc#L600-L608

**Comment by kjvbrt** [comment]
looks good :)

**Pull request merged**